pytorch_lightning/callbacks/pt_callbacks.py

```diff
@@ -149,13 +165,14 @@ def check_metrics(self, logs):
 
         return True
 
-    def on_train_begin(self, logs=None):
+    def on_train_begin(self):
         # Allow instances to be re-used
         self.wait = 0
         self.stopped_epoch = 0
         self.best = np.Inf if self.monitor_op == np.less else -np.Inf
 
-    def on_epoch_end(self, epoch, logs=None):
+    def on_epoch_end(self):
+        logs = self.trainer.callback_metrics
```

Comment:
pls add assert that the `self.trainer` is not `None`

Reply:
Done.